docs/content/concepts/assets/asset-observations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Asset Observations | Dagster
+description: Dagster provides functionality to record metadata about assets.
+---
+
+# Asset Observations
+
+An asset observation is an event that records metadata about a given asset.
+
+## Relevant APIs
+
+| Name                                                    | Description                                                          |
+| ------------------------------------------------------- | -------------------------------------------------------------------- |
+| <PyObject module="dagster" object="AssetObservation" /> | Dagster event indicating that an asset's metadata has been recorded. |
+| <PyObject module="dagster" object="AssetKey" />         | A unique identifier for a particular external asset                  |
+
+## Overview
+
+<PyObject module="dagster" object="AssetObservation" /> events are used to communicate
+to Dagster that metadata has been recorded about a given asset. Asset observation
+events can be yielded at runtime within ops and assets. An asset must be defined
+using the <PyObject object="asset" decorator /> decorator or have existing materializations
+in order for its observations to be displayed.
+
+## Yielding an AssetObservation from an Asset
```

Comment:
Because ops / jobs are the APIs that we forefront on the rest of the site, I think we should have the primary examples here be in terms of ops, instead of software-defined assets.